setup.py

```diff
@@ -15,6 +15,7 @@
       license='BSD 3-clause "New" or "Revised" License',
       packages=['pipeline_tools'],
       install_requires=[
+          'arrow>=0.12.1',
           'google-auth>=1.6.1,<2',
           'google-cloud-storage>=1.8.0,<2',
```

Comment:
Is it intentional that this doesn't match the version text in `requirements.txt`? I'm not familiar with python but I'd hope there would be a way to only specify the dependency list once, instead of copying it in both places.

Reply:
This has definitely caused confusion in the past...but currently we use the `requirements.txt` to set-up our local development environment for pipeline-tools, and the `setup.py` requirements get used when the package is installed in the Dockerfile. They should be the same but it's harder to maintain when it's copied in two places.